monai/data/dataloader.py

```diff
@@ -33,11 +33,11 @@ class DataLoader(torch.utils.data.DataLoader):
         num_workers (int, optional): how many subprocesses to use for data
             loading. ``0`` means that the data will be loaded in the main process.
             (default: ``0``)
-        pin_memory (bool, optional): If ``True``, the data loader will copy Tensors
+        pin_memory: If ``True``, the data loader will copy Tensors
```

Comment:
why is this corresponding signature not updated? some compatibility issue?

Reply:
Thanks, we just made the fix.